apps/browser-tests/src/App.tsx

```diff
@@ -105,15 +110,25 @@ function Transactions({
           ],
         });
       } else if (provider === "magiceden") {
-        signPsbtResponse = await magiceden.signPsbt(psbt.toPSBT(), {
-          network,
-          inputsToSign: [
-            {
-              address: inputAddressInfo.address,
-              signingIndexes: [0],
-            },
-          ],
-        });
+        const meWallet = satsConnectWallets!.find(
+          (wallet) => wallet.name === "Magic Eden",
+        );
+        const magicEdenWalletProvider = (meWallet as MagicEdenWallet)?.features[
+          "sats-connect:"
+        ]!.provider;
```

Comment:
apply the type to `meWallet` above directly.

accessing features.provider might crash, verify that the keys exists before doing so - is there a utility exposed by wallet-standard?

Reply:
Replaced with utility function.